.gitignore

```diff
@@ -17,6 +17,7 @@
 [Bb]inaries/
 [Bb]in/
 [Oo]bj/
+[Bb]enchmarkDotNet.[Aa]rtifacts/
```

Comment:
Nit: why is benchmarkDotNet.artifacts ignored but not for example benchmarkdotnet.artifacts? Maybe this doesn't need the [] character groups at all (the generated folder should always have the correct casing)?

Reply:
Was just following the pattern here. You can make the same case for `bin` and `obj` as they're always emitted with lower case. Overall though it does seem quite silly. Will simplify this entry at least.